gui/widgets/qopenglmousetracker.cpp

```diff
@@ -81,6 +81,17 @@ void QOpenGLMouseTracker::renderScene(std::optional<float> lineWidth) {
   }
 }
 
+void QOpenGLMouseTracker::updateAllClippingPlanes(
+    std::unique_ptr<rendering::ShaderProgram> &program) const {
```

Comment:
does this need to take program as an argument - isn't it always going to use the member variable program?